src/legacy/core_plugins/kibana/public/dashboard/panel/_dashboard_panel.scss

```diff
@@ -158,6 +158,15 @@
   }
 }
 
+@media (hover: none) {
+  /**
+   * Always show the option menu button on devices that don't support hovering (like most touch devices).
+   */
+  .dshPanel_optionsMenuButton {
+    opacity: 1;
```

Comment:
Are you sure we need to do this? Displaying the button at all times for all charts adds a lot of clutter to the screen and doesn't work well when panel headers are hidden.

<img width="50%" alt="Screen Shot 2019-04-11 at 10 54 18 AM" src="https://user-images.githubusercontent.com/549577/55968851-cd1bd780-5c4a-11e9-8eea-7f083f537e0e.png">


<img width="50%" alt="Screen Shot 2019-04-11 at 10 54 11 AM" src="https://user-images.githubusercontent.com/549577/55968763-a3fb4700-5c4a-11e9-8c25-dc3a38949d9a.png" >

_tablet size_


----

I checked in Chrome on Android and Safari on iOS and the both still support the `:hover` by acting as `:focus`. This also works for the legend toggle as well as the panel options button.

<img src="https://d.pr/free/i/l5jlje+" width="50%" />

Reply:
The problem that I see here, is that this is completely invisible to the user. Also depending on the chart a lot of the panel area might be a click area of the chart, so you basically need to be able to touch the small title bar on top, for getting in that focus state to then click that button. I agree that it's putting a bit more noise onto it, but I think not making that functionality at all visible or intuitive to use, is neither a good solution.

Reply:
I totally understand the thought behind and most of the time you do want to surface hover-only actions. However, in this case the main purpose of the dashboard is to view the visualizations and that little extra bit of clutter can really detract from that. Also, since most people do tend to click on thing to find the interactions, I don't **think** it'll be a discoverability problem, and before we implement such a visual change, I'd like to wait to see if we start getting a lot of feedback on it.

> Also depending on the chart a lot of the panel area might be a click area of the chart, so you basically need to be able to touch the small title bar on top

This actually is not true as the `~:focus` state will also be triggered when something inside the panel is focused. Example is with maps:

<img src="https://d.pr/free/i/mXJNlf+" width="50%" />

Reply:
Okay I'll leave the decision on that to you, and will revert that change :-)

Reply:
Thanks! We'll monitor any feedback that comes back for mobile support in general.